breadcord/bot.py

```diff
@@ -177,10 +177,20 @@ async def load_wrapper(module_id: str) -> None:
                 return
             await self.modules.get(module_id).load()
 
-        await asyncio.gather(*(
-            load_wrapper(module)
-            for module in self.settings.modules.value
-        ))
+        modules: list[str] = self.settings.modules.value  # type: ignore
+        unduped: list[str] = []
+        for module in modules:
+            if module not in unduped:
+                unduped.append(module)
+
+        if len(modules) != len(unduped):
+            _logger.warning(
+                f'Duplicate module entries found in settings. '
+                f'Removing {len(modules) - len(unduped)} duplicate(s).',
+            )
+            modules = self.settings.modules.value = unduped
+
+        await asyncio.gather(*(load_wrapper(module) for module in modules))
```

Comment:
I feel like this would be cleaner as:
```suggestion
        await asyncio.gather(*map(load_wrapper, modules))
```
Not that it matters though!